pyfolio/utils.py

```diff
@@ -192,6 +192,11 @@ def print_table(table, name=None, fmt=None, header_rows=None):
         Formatter to use for displaying table elements.
         E.g. '{0:.2f}%' for displaying 100 as '100.00%'.
         Restores original setting after displaying.
+    formatters : list or dict, optional
```

Comment:
should we combine this and `fmt` above, or rename one of them? it's slightly confusing to have them both

Reply:
From what I can tell, all the existing call sites that pass `fmt` are passing `'{0:.2f}'` to format float values. `pd.DataFrame.to_html()` accepts a `float_format` arg, so what would you think of  renaming `fmt` to `float_format`, and just forwarding that to `to_html()`? The call sites would then pass `'{0:.2f}.format'`, and the behavior should be the same.

Reply:
👍